packages/bpk-react-utils/src/cssModules.js

```diff
@@ -16,5 +16,11 @@
  * limitations under the License.
  */
 
-export default (styles = {}) => className =>
-  styles[className] ? styles[className] : className;
+export default (styles = {}) => (...classNames) =>
```

Comment:
can now be used like this:

`
getClassNames('some-class', classNames, disabled && 'disabled')
`

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/359644fb-c5df-4412-9bc7-ae838696ef14-7fa1897d-dae4-4909-a4ec-c165763ceef8-v1.png?transparent=1&palette=1&width=246)